ci/nctl_upgrade.sh

```diff
@@ -148,6 +148,11 @@ function start_upgrade_scenario_9() {
     nctl-exec-upgrade-scenario-9
 }
 
+function start_upgrade_scenario_11() {
+    log "... Starting Upgrade Scenario 11"
+    nctl-exec-upgrade-scenario-11
+}
+
```

Comment:
Can this go after `function start_upgrade_scenario_10()`?

Reply:
For sure. Updated in e7446a6c614e9f50ac7ac5cf367e0074f1c40c5e